templates/swagger-ui.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
`templates/swagger-ui.html` should remove as `drf_spectacular` included

Reply:
It can not be dropped